src/main/java/org/dinosaur/foodbowl/domain/post/api/PostController.java

```diff
@@ -0,0 +1,31 @@
+package org.dinosaur.foodbowl.domain.post.api;
+
+import lombok.RequiredArgsConstructor;
+import org.dinosaur.foodbowl.domain.post.application.PostService;
+import org.dinosaur.foodbowl.domain.post.dto.response.PostThumbnailResponse;
+import org.dinosaur.foodbowl.global.dto.PageResponse;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort.Direction;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/posts")
+@RestController
+public class PostController {
+
+    private final PostService postService;
+
+    @GetMapping("/thumbnails")
+    public ResponseEntity<PageResponse<PostThumbnailResponse>> findThumbnailsInProfile(
+            @RequestParam Long memberId,
+            @PageableDefault(size = 18, page = 0, sort = "createdAt", direction = Direction.DESC) Pageable pageable
+    ) {
+        PageResponse response = postService.findThumbnailsInProfile(memberId, pageable);
```

Comment:
raw 타입을 사용하고 있어 warning이 발생할 것 같습니다 !

```suggestion
PageResponse<PostThumbnailResponse> response = postService.findThumbnailsInProfile(memberId, pageable);
```

는 어떨까요?

Reply:
놓쳤던 부분이네요 감사합니다 👍